### ISSUE TITLE
Trying to export symbol as SVG

### ISSUE THREAD

**Issue by billhannah** [avocode/avocode#1347]
IMPORTANT: Please try to describe how did it happen.
It seems the symbol wasn't synced when trying to export.
-----
App v. `2.13.1`
Avocode UID: 72062
```
TypeError: Promise.reject is not a constructor
    at e.exportToSvg (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:55:30052)
    at e.exportLayers (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:55:30357)
    at e._exportLayersToDirectory (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:52:9075)
    at e._exportRuleVariantToDirectory (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:52:7964)
    at wrapper (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/lodash/lodash.js:5232:19)
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:3018:46
    at List.__iterate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:2208:13)
    at IndexedIterable.mappedSequence.__iterateUncached (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:3017:23)
    at seqIterate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:606:16)
    at IndexedIterable.IndexedSeq.__iterate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:322:14)
    at IndexedIterable.toArray (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:4260:23)
    at new List (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:2067:62)
    at reify (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:3572:37)
    at List.map (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:4403:14)
    at e._exportRuleGroupToDirectory (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:52:7688)
    at wrapper (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/lodash/lodash.js:5232:19)
    at Array.map (native)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:52:5416
```

**Comment by vojtatranta** [comment]
Thank you!

Thats a bug, we will fix it.

**Comment by jakubzitny** [comment]
This should be fixed in Avocode `2.13.2` we've just released. Let us know if this problem persists or if you have any other issues with it.

Thanks.
